src/ptvsd/_vendored/pydevd/third_party/cython_json.py

```diff
@@ -0,0 +1,293 @@
+import Cython
```

Comment:
Note: this is actually unrelated to the fix (just synchronizing with the `pydevd` source code -- this is not really used in the debugger).